### PR TITLE
Mj wip frd automation

### PR DESCRIPTION
**Pull request by Mohit344**
Done modification in extent report and test base.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -118,18 +101,7 @@
...
-        <version>5.0.0</version>
-    </dependency>
-    <dependency>
-        <groupId>org.apache.poi</groupId>
-        <artifactId>poi</artifactId>
-        <version>5.0.0</version>
-    </dependency>
+        </dependency>
         <dependency>
             <groupId>com.aventstack</groupId>
```
Comment (rajmyil) [inline pom.xml:106]: Remove the aventStack dependency. extent report should have the relevantcode dependency alone

File: pom.xml [hunk 2]
```diff
@@ -155,44 +127,17 @@
...
-        </plugins>
-    </build>
+        <!-- https://mvnrepository.com/artifact/com.relevantcodes/extentreports -->
+        <dependency>
+            <groupId>com.relevantcodes</groupId>
+            <artifactId>extentreports</artifactId>
+            <version>2.41.2</version>
+        </dependency>
+        <dependency>
+            <groupId>com.aventstack</groupId>
```
Comment (rajmyil) [inline pom.xml:137]: Remove aventstack dependency

File: src/main/java/com/frd/automation/utils/HelperMethods.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.frd.automation.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class HelperMethods {
```
Comment (rajmyil) [inline src/main/java/com/frd/automation/utils/HelperMethods.java:6]: @arunatm2017 HelperMethods is fine as name? Can we rename as CommonUtils?

**Review by rajmyil — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Mohit344) [reply pom.xml:106]: sure i remove aventstack dependency

File: pom.xml (hunk 2 above)
Comment (Mohit344) [reply pom.xml:137]: sure i will remove aventstack dependency

File: src/main/java/com/frd/automation/utils/HelperMethods.java (hunk 3 above)
Comment (arunatm2017) [reply src/main/java/com/frd/automation/utils/HelperMethods.java:6]: Yes **CommonUtils** is fine.

Comment (Mohit344) [reply src/main/java/com/frd/automation/utils/HelperMethods.java:6]: ok i will change it to CommonUtils

**Review by rajmyil — APPROVED** [review]

**Review by arunatm2017 — APPROVED** [review]

**Review by rajmyil — APPROVED** [review]

**Pull request merged**